src/python/py_vlm_pipeline.cpp

```diff
@@ -267,6 +270,23 @@ void init_vlm_pipeline(py::module_& m) {
             py::arg("streamer") = std::monostate(), "streamer",
             (vlm_generate_docstring + std::string(" \n ")).c_str()
         )
+        .def(
+            "generate",
+            [](ov::genai::VLMPipeline& pipe,
+                const std::string& prompt,
+                const std::vector<ov::Tensor>& videos,
```

Comment:
why don't we have `images` argument here as well? Please elaborate how we pass both `images` and `videos` into the same generate.